test/node/timestamp.test.ts

```diff
@@ -22,9 +22,9 @@ describe('Timestamp', () => {
       // We do expect it to work so that round tripping the Int32 instance inside a Timestamp works
       new BSON.Timestamp({ t: new BSON.Int32(0x7fff_ffff), i: new BSON.Int32(0x7fff_ffff) })
     ];
-
     for (const timestamp of table) {
-      expect(timestamp).to.have.property('unsigned', true);
+      expect(timestamp.t).to.be.gte(0);
+      expect(timestamp.i).to.be.gte(0);
```

Comment:
Is this a useful check?

Reply:
Should we add one more test to the table that passed in `-1` for `t`/`i`?

Reply:
We have tests that check that it throws when passing in a negative number for either `t` or `i`